src/serac/physics/tests/parameterized_thermomechanics_example.cpp

```diff
@@ -88,8 +86,15 @@ TEST(Thermomechanics, ParameterizedMaterial)
     serac::StateManager::setMesh(std::move(mesh));
   }
 
-  SolidMechanics<p, dim, Parameters<H1<p>, H1<p> > > simulation(default_static_options, GeometricNonlinearities::On,
-                                                                "thermomechanics_simulation");
+  serac::NonlinearSolverOptions nonlin_options{.nonlin_solver  = NonlinearSolver::KINPicard,
+                                               .relative_tol   = 1.0e-12,
+                                               .absolute_tol   = 1.0e-12,
+                                               .max_iterations = 30,
+                                               .print_level    = 1};
```

Comment:
Just to make sure the Picard option is working. I'm happy to move it to another test if you have ideas.

Reply:
Do we already have a dedicated set of tests for just the `EquationSolver` class? That would be a more natural place to figure out if its different options are working properly. 

I think files labeled with "example" should focus more on demonstrating how to use serac effectively, instead of testing solver options.